src/main/kotlin/com/cognifide/gradle/aem/AemConfig.kt

```diff
@@ -86,6 +86,17 @@ open class AemConfig(project: Project) : Serializable {
     var deployParallel: Boolean = false
 
     /**
+     * Determines if packages to be installed by e.g satisfy or deploy task, could contain subpackages.
+     * If packages are exclusive then asking for installed packages will be done only once.
+     *
+     * Exclusive means that particular package installation cannot be a cause of that next package
+     * to be installed will be already installed.
+     */
+    @Incubating
+    @Input
+    var deployExclusive: Boolean = true
```

Comment:
As far as I can see, this flag will be used only for satisfy and it makes sense only for satisfy. The description may not be accurate then. Second, it's pretty hard to grasp.

The flag changes the behaviour of satisfy task, we should abstract from the undelying cause for which user may want to change the value of the flag (e.g. subpackages), because the reason may be different. Or user does not bother having this package being reinstalled.

    I'd go with: 
 ```
    * aemSatisfy is a lazy task, which means it won't install package that is already installed. 
     * By default, `aemSatisfy` task will retrieve package list from AEM only once and check 
     * all aemSatisfy packages against it.
     * 
     * This flag can change that behavior, so the package list can be refreshed after each package installation. 
     * 
     * This feature flag may be useful if particular package contains subpackages, which may be listed 
     * as a separate entities on aemSatisfy list and we don't want them to be reinstalled. This rarely 
     * should happen in a project.
     */
    @Incubating
    @Input
    var refreshPackageListAfterEachInstall: Boolean = false;
```

Reply:
Generally agree.

About messages, some plugin users might have get used to current messages. I am not sure if there will be value added if I change them right now.

The flag is right for satisfy. I was wondering if it could be used e.g when `build aemUpload aemInstall` will be called separately, but this is almost not the right case, because `aemDeploy` or `aemBuild` should be used which does not require package list retrieval. 

I observed a gain, for few packages that I have on satisfy list, I reduced aemSatisfy time from 8 to 3 seconds so I am sure that the change is valueable.

Reply:
I decided to change that flag name to `satisfyRefreshing` (easier to remember), flag will be very rarely used I think so no matter to  spend more time on it.